Library/Homebrew/extend/os/linux/development_tools.rb

```diff
@@ -19,5 +19,10 @@ def locate(tool)
     def default_compiler
       :gcc
     end
+
+    def build_system_info
+      glibc_version = Formula["glibc"].any_installed_version&.pkg_version || OS::Linux::Glibc.system_version
```

Comment:
`pkg_version` would actually be this:

```suggestion
      glibc_version = Formula["glibc"].any_installed_version || OS::Linux::Glibc.system_version
```

`any_installed_version` returns a `PkgVersion`, and doing `.version` on that is what strips the revision and returns a regular `Version`.

Reply:
Thank you, Bo!